frontend/src/bundles/common/components/bar-chart/bar-chart.tsx

```diff
@@ -0,0 +1,43 @@
+import {
+    BarElement,
+    CategoryScale,
+    Chart as ChartJS,
+    LinearScale,
+    Tooltip,
+} from 'chart.js';
+import React from 'react';
+import { Bar } from 'react-chartjs-2';
+
+import { BarColor, BarLabel } from '../../enums/enums';
+import { dateHelper } from '../../helpers/helpers';
+import styles from './bar.module.scss';
+import { options } from './config-bar';
+
+ChartJS.register(CategoryScale, LinearScale, BarElement, Tooltip);
+
+type DataObject = { date: string; value: number };
+type ChartProperties = {
+    income: DataObject[];
+    outcome: DataObject[];
+};
+
+const Chart: React.FC<ChartProperties> = ({ income, outcome }) => {
+    const data = {
+        datasets: [
+            {
+                label: BarLabel.OUTCOME_LABEL,
+                data: dateHelper(outcome),
+                backgroundColor: BarColor.OUTCOME_COLOR,
+            },
+            {
+                label: BarLabel.INCOME_LABEL,
+                data: dateHelper(income),
+                backgroundColor: BarColor.INCOME_COLOR,
+            },
```

Comment:
I think it's better to replace 'income' and 'outcome' props with one datasets prop

Reply:
fix in [https://github.com/BinaryStudioAcademy/bsa-winter-2022-2023-smartspend/pull/32/commits/81e9f408873f7c552d42647c617ef8337b7546ab](url)